### ISSUE TITLE
Packages available: Not Found

### ISSUE THREAD

**Issue by mounta11n** [holzschu/a-Shell-commands#44]
Hi, whenever I try to install a package or to list the available packages, it returns that no packages are found.
I tried a forked repo as $PKG_SERVER but same error. I currently have set it back to `$PKG_SERVER=https://github.com/holzschu/a-Shell-commands`.

What could be the reason for this behavior? I have to mention that I did not have had all the time this error, since I already have installerd some packages in the past.

Would be very happy for any help

**Comment by mounta11n** [comment]
Okay I tried the command `$ unsetenv PKG_SERVER` and now it works again. Now `pkg search` will list all available packages, but unfortunately it still doesn‘t work if I set another $PKG_SERVER in my .bashrc, even if it is configured back to `https://github.com/holzschu/a-Shell-commands`. In this case I again have to run `$ unsetenv PKG_SERVER`.

Therefore I will leave this issue opened as it could really be a bug or something.
